src/Middleware/Spa/SpaProxy/src/SpaHostingStartup.cs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+
+[assembly: HostingStartup(typeof(Microsoft.AspNetCore.SpaProxy.SpaHostingStartup))]
```

Comment:
Why?

Reply:
This is a development time only package that we wire-up in launchSettings.json and that takes care of launching the angular/react proxy server when you do `dotnet run` it's not used in the published app.

Reply:
You can find all the details [here](https://gist.github.com/javiercn/b28cb0ea0912de8313e0fd0cff9021d9#file-spec-md)